resource-managers/mesos/src/main/scala/org/apache/spark/deploy/rest/mesos/MesosRestServer.scala

```diff
@@ -153,6 +156,20 @@ private[mesos] class MesosSubmitRequestServlet(
         handleError(s"Received message of unexpected type ${unexpected.messageType}.")
     }
   }
+
+  private[mesos] def validateLabelFormat(properties: Map[String, String]
+                                       , propertyNames: String*): Unit = {
+    propertyNames.foreach { name =>
```

Comment:
as mentioned above, if the labels don't change, I think we can remove the `propertyNames` arg and do a simple
```
List("spark.mesos.network.labels", "spark.mesos.task.labels", ...).foreach { =>

Reply:
changed as suggested